style-guide/blog-guidelines.rst

```diff
@@ -0,0 +1,198 @@
+----------------------------------------
+
+Before you start writing, consider the following questions:
+
+- What is the goal of this developer blog? Describe it in two sentences or
+  less.
+
+  - Is it to be helpful and answer a question or offer how-to information?
+  - Is it to provide “thought leadership” or show our expertise in an area?
+    If so, describe what you’d like to convey.
+  - Is it something else altogether? If so, what? It’s possible that your
+    idea might be better presented as an article on the How-To support site
+    at https://support.rackspace.com/ or as a blog with a marketing flavor
+    published at https://blog.rackspace.com/.
+
+- What is the significance of the content of the blog? Is it how to use a new
+  offering? How is it helpful to customers? Does it frame our
+  expertise is a new way?
+- Explain how the content of the blog aligns with, supports, or strengthens
+  Rackspace’s current business priorities and strategy.
+- Who is the target audience for this blog?
+- What specific customer problems or pain points does the blog address?
+  Provide names of customers willing to be references in support of the post.
+- Does the blog describe how to use something that Rackspace provides that
+  it does better than the competition or that no other competitor offers? Does
+  the blog provide quantifiable and defensible information (such as
+  performance metrics) showing the Rackspace edge over the competition? If
+  yes, explain.
+
+
+Blog writing suggestions
+------------------------
+
+When you write your content, we recommend the following rules of thumb:
+
+-  Length: Rackspace blog posts generally run between 500-1,000 words, but let
+   the content be your guide. Take the space you need to tell your story, but
+   don’t pad it unnecessarily. Going long is okay, too, as long as it’s worth
+   reading.
+
+-  Tools: Use the word processor or text editor of your choice to write the
+   blog.
+
+-  General format:
+
+   - An attention-grabbing opening, known in the news business as a “lede”
+
+   - A “nut graf” - a sentence or two letting readers know what you’re going
+     to be talking about and why it matters
+
+   - Some background information or context, such as the history of a product
+
+   - A conclusion (and a call to action, if appropriate) with links and contact
+     information
+
+- Consistency: Be as consistent as possible in style with other blogs on the
+  Rackspace Developer Blog site.
+
+- Bullet points: Bullet points are useful for listing features.
+
+- Images:
+
+  - Any image included in the blog post should contribute to the content of
+    that specific blog.
+  - Any image pulled from another source should provide a reference to the
+    source directly underneath the image.
+  - Add social media icons with their links for twitter, Facebook, and
+    LinkedIn at the bottom of the blog post. (Information Development provides
+    a template for this that you can add to your blog.)
```

Comment:
- Images should directly support or illustrate blog content.  For example, a screen shot, diagram or table that illustrates the concept or technique being described.